lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -227,6 +227,8 @@ class DocumentImpl extends NodeImpl {
     }
 
     this._contentType = privateData.options.contentType;
+    this._encoding = privateData.options.encoding;
```

Comment:
Or maybe should it be "UTF-8" if the string content is provided by the user and "windows-1252" if the content is decoded from jsdom.